### PR TITLE
Added AP_WINCH options to Custom-Build server

### PR DESCRIPTION
**Pull request by adityaomar3**
AP_WINCH build options have been added to custom-build server
Please review the addition of feature in list.

### CODE REVIEW

**Comment by peterbarker** [comment]
> I have been reading code and features for custom-build server and found out precland sitl and sitl gazebo have been done but not added to custom build list. Please review the addition of feature in list.

Currently the custom build server only builds STM32 ('chibios-based) boards, so the SITL drivers aren't relevant.

File: Tools/scripts/build_options.py [hunk 1]
```diff
@@ -347,7 +347,8 @@ def __init__(self,
     Feature('Precision Landing', 'PrecLand', 'AC_PRECLAND_ENABLED', 'Enable Precision Landing support', 0, None),
     Feature('Precision Landing', 'PrecLand - Companion', 'AC_PRECLAND_COMPANION_ENABLED', 'Enable Companion-Supported Precision Landing support', 0, "PrecLand"),  # noqa
     Feature('Precision Landing', 'PrecLand - IRLock', 'AC_PRECLAND_IRLOCK_ENABLED', 'Enable IRLock-Supported Precision Landing support', 0, "PrecLand"),  # noqa
-
+    Feature('Precision Landing', 'PrecLand - SITL_GAZEBO', 'AC_PRECLAND_SITL_GAZEBO_ENABLED', 'Enable SITL-GAZEBO Precision Landing support', 0, "PrecLand"),
```
Comment (peterbarker) [inline Tools/scripts/build_options.py:350]: I would imagine that trying to compile these into stm32-based boards won't work at the moment, and probably wouldn't be useful if they did compile in :-)

**Comment by peterbarker** [comment]
@adityaomar3 why don't you have a go at adding the winch backends to the build options?

File: Tools/scripts/build_options.py (hunk 1 above)
Comment (adityaomar3) [reply Tools/scripts/build_options.py:350]: Yes, That's totally makes sense, if we are doing it for stm32 then we wont be needing the sitl builds.

**Comment by adityaomar3** [comment]
@peterbarker can  you  help me with the failing checks, it is failing checks for some boards, I think winch is not configured for them?

**Review by peterbarker — APPROVED** [review]
I've updated this to be correct and put it under"Payload" so we don't blow out the number of sections we have in the interface.

**Comment by peterbarker** [comment]
Merged this, thanks @adityaomar3

**Pull request merged**